ui/src/components/NewsCarousel/index.tsx

```diff
@@ -0,0 +1,56 @@
+import {Carousel} from 'react-responsive-carousel';
+import 'react-responsive-carousel/lib/styles/carousel.min.css';
+import {dummyNewsImg, loginArrow} from '../../assets';
+import styles from './styles.module.css';
+
+const data = [
+  {
+    title: 'Registrations Open',
+    clubName: 'The Programming Club',
+    description:
+      'Bayerische Motoren Werke AG, commonly referred to as BMW is a German multinational corporate manufactu',
+    imgUrl: dummyNewsImg,
+  },
+  {
+    title: 'Registrations Open',
+    clubName: 'The Programming Club',
+    description:
+      'Bayerische Motoren Werke AG, commonly referred to as BMW is a German multinational corporate manufactu',
+    imgUrl: dummyNewsImg,
+  },
+  {
+    title: 'Registrations Open',
+    clubName: 'The Programming Club',
+    description:
+      'Bayerische Motoren Werke AG, commonly referred to as BMW is a German multinational corporate manufactu',
+    imgUrl: dummyNewsImg,
+  },
+];
+
+export const NewsCarousel: React.FC = () => {
+  return (
+    <Carousel
```

Comment:
When the data is empty it should should show no events yet at the center.

Reply:
When the data is empty, this component does not render anything.

Reply:
I see. Yeah, I think we can do away with this element if there are no highlighted events.